wiki/Installation/macOS/en.md

```diff
@@ -1,20 +1,43 @@
 # Installation on macOS
 
-This page will roughly tell you how to get osu! working on your macOS device.
+This page will roughly guide you on getting osu! working on your macOS device.
 
 ## Minimum system requirements
 
 - A decent computer running macOS.
 
+## osu!lazer
```

Comment:
if anything, this header should be below Troubleshooting

Reply:
I got the impression that peppy wanted to make the osu!lazer client the more obviously recommended one (https://github.com/ppy/osu-wiki/pull/4313) which is why I left it as the first heading but yeah I'll put it at the bottom